Demos/freeze/cython_freeze

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env python
+"""
+Create a C file for embedding one or more Cython source files.
```

Comment:
I dislike the idea of duplicating files in the source tree, even just for demo/test purposes. If the script needs fixing, we should fix the script, not start having to maintain two of them.